beta/examples/tensorflow/segmentation/evaluation.py

```diff
@@ -235,7 +234,7 @@ def main(argv):
     if 'train' in config.mode or 'test' in config.mode:
         run_evaluation(config)
 
-    if 'export' in config.mode:
+    elif 'export' in config.mode:
```

Comment:
I think the user should be able to test and then export the model in one run.

Reply:
This opportunity is implemented here
https://github.com/openvinotoolkit/nncf/blob/f8231f9c2f054301adc51dcb1ef4d53ee66a6bd0/beta/examples/tensorflow/segmentation/evaluation.py#L184-L187

These changes are needed to avoid saving the model twice.

Reply:
I've  caught